examples/amp-story/doubleclick.html

```diff
@@ -42,7 +42,7 @@
               {
                 "ad-attributes": {
                   "type": "doubleclick",
-                  "data-slot": "/30497360/samfrank_native_v2_a4a"
+                  "data-slot": "/30497360/a4a/amp_story_dfp_example"
```

Comment:
i suppose this is another valid slot ID?

Reply:
its the only valid slot id :) The other one was hacky.